server/src/main/scala/drt/server/feeds/cirium/CiriumFeed.scala

```diff
@@ -98,6 +98,31 @@ object CiriumFeed {
     )
   }
 
+  private def extractEstChox(f: CiriumFlightStatus) = {
+    (f.operationalTimes, f.flightDurations) match {
+      case (o, _) if o.estimatedGateArrival.isDefined => o.estimatedGateArrival.map(_.millis)
+      case (o, Some(d)) if o.estimatedRunwayArrival.isDefined && d.scheduledTaxiInMinutes.isDefined =>
+       o.estimatedRunwayArrival
```

Comment:
Could refactor to use a for yield - might be clearer, but I'm not actually sure;
```
for {
  runwayArrival <- o.estimatedRunwayArrival
  taxiMinutes <- o.scheduledTaxiInMinutes
} yield SDate(runwayArrival).addMinutes(taxiMinutes).millisSinceEpoch
```